installer/manifests/keptn/values.yaml

```diff
@@ -1227,6 +1226,9 @@ logLevel: "info"
 ## @param prefixPath URL prefix for all Keptn URLs
 prefixPath: ""
 
+## @param keptnUsername for Keptn Bridge login
+keptnUsername: ""
```

Comment:
Please put this under the `bridge` values

Reply:
Thanks much @mowies for your quick reply! I loved that.